### ISSUE TITLE
salt boostrap on Arch Linux

### ISSUE THREAD

**Issue by techdragon** [saltstack/salt-bootstrap#139]
I think salt needs an override option of some kind "--skip-pgp" or something.

I need to manually do this in order to bypass the issues i find with a few of the AUR dependencies salt's bootstrap pulls in.

I can do it manually but I'm sure I'm not the only one who wouldnt mind an override to not have to do it manually.

**Comment by s0undt3ch** [comment]
I noticed this issue yesterday while trying to bootstrap and Arch EC2 instance.
I though it could be a temporary repository issue.

Need to bring the big guns in, @thatch45, @techhat, what should be the proper approach to?

```
:: File /var/cache/pacman/pkg/python2-msgpack-0.3.0-1-x86_64.pkg.tar.xz is corrupted (invalid or corrupted package (PGP signature)).
Do you want to delete it? [Y/n] error: failed to commit transaction (invalid or corrupted package)
```

**Comment by s0undt3ch** [comment]
This has been fixed in ce1cc43
